src/components/molecules/Visualizer/Plugin/hooks.ts

```diff
@@ -67,7 +71,7 @@ export default function ({
     onMessage,
     onPreInit,
     onDispose,
-    autoResize,
+    autoResizeDirection,
```

Comment:
```suggestion
    actualAutoResize,
```

How about "actualAutoResize"?